client/dashboard/sites/add-new-site/index.tsx

```diff
@@ -0,0 +1,139 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { WordPressLogo, JetpackLogo } from '@automattic/components';
+import { formatNumber } from '@automattic/number-formatters';
+import {
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	__experimentalText as Text,
+	Button,
+} from '@wordpress/components';
+import { __, sprintf } from '@wordpress/i18n';
+import { download, reusableBlock, Icon } from '@wordpress/icons';
+import clsx from 'clsx';
+import devSiteBanner from 'calypso/assets/images/a8c-for-agencies/dev-site-banner.svg';
+import { preventWidows } from 'calypso/lib/formatting';
+import Column from './column';
+import MenuItem from './menu-item';
+import './style.scss';
+
+const TRACK_SOURCE_NAME = 'sites-dashboard';
+
+const wordpressClick = () => {
+	recordTracksEvent( 'calypso_sites_dashboard_new_site_action_click_add' );
+	recordTracksEvent( 'calypso_sites_dashboard_new_site_action_click_item', {
+		action: 'wordpress',
+	} );
+};
+const jetpackClick = () => {
+	recordTracksEvent( 'calypso_sites_dashboard_new_site_action_click_jetpack' );
+	recordTracksEvent( 'calypso_sites_dashboard_new_site_action_click_item', {
+		action: 'jetpack',
+	} );
+};
+const migrateClick = () => {
+	recordTracksEvent( 'calypso_sites_dashboard_new_site_action_click_item', {
+		action: 'migrate',
+	} );
+};
+const importClick = () => {
+	recordTracksEvent( 'calypso_sites_dashboard_new_site_action_click_import' );
+	recordTracksEvent( 'calypso_sites_dashboard_new_site_action_click_item', {
+		action: 'import',
+	} );
+};
+const offerClick = () => {
+	recordTracksEvent( 'calypso_sites_dashboard_new_site_action_click_item', {
+		action: 'offer',
+	} );
+};
+
+function AddNewSite() {
+	return (
+		<HStack alignment="flex-start" style={ { padding: '16px' } } spacing={ 4 }>
+			<Column title={ __( 'Add new site' ) }>
+				<MenuItem
+					icon={ <WordPressLogo /> }
+					title={ __( 'WordPress.com' ) }
+					description={ preventWidows(
+						__( 'Build and grow your site, all in one powerful platform.' )
+					) }
+					onClick={ wordpressClick }
+					href="/start?source=sites-dashboard&ref=new-site-popover"
```

Comment:
Since we're using regular links now for all these links instead of `page` I'm wondering if:

 - Should we be using "page" for the links that go outside the dashboard v2 (I don't really want to import page tbh)
 - Should we call "localizeUrl" on these links?

cc @jsnajdr

Reply:
What's a good reason to use `page`? What we have here seems like a good compromise.

And I think we generally localize these URLs, but at the same time, I believe `localizeUrl` requires a full URL and won't work with a relative path.

Reply:
Using `page` will ensure a client side navigation if I import this component in the existing dashboard (to avoid duplication). But maybe it's ok to avoid it here.

Reply:
You don't need `page` to ensure good behavior of "native" links like `<a href="...">`. If the component is used in existing Calypso UI that uses `page`, there is a global `onclick` listener (on `document`) that intercepts clicks on links and handles them using `page`:

https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-router/src/index.js#L725

There is an opposite problem though: TanStack Router doesn't have such a global listener (it's a non-explicit somewhat intrusive hack, after all) and if you want your `<a href>` links to be handled by the router, you need to use the router's own `<Link>` component.

Reply:
In this particular case, it's not a problem because all these links go outside the React Router and we don't really want/need client side navigation coming from v2.